src/tstester/servicewrapper.ts

```diff
@@ -22,13 +19,21 @@ class BaseServiceEventTrigger extends EventWithHoldAdapter<JDEvent> {
     }
 }
 
+export class ServiceHold extends BaseServiceEventTrigger {
+    protected processHold(data: JDEvent) {
+        throw new Error(
+            `service ${this.service.name} got event ${data.code} (${data.name}) when hold asserted`
+        )
+    }
+}
+
 // Event that fires on a matching event code from the specified service
 class BaseServiceAnyEventTrigger extends BaseServiceEventTrigger {
```

Comment:
same comment about base

Reply:
Not completely sure what you're referring to, but added and updated some comments.